manager/manager/launcher/launcher_drones_ros2.py

```diff
@@ -22,22 +22,13 @@ def run(self, callback):
         # expand variables in configuration paths
         world_file = os.path.expandvars(self.launch_file)
 
-        # Launching MicroXRCE and Aerostack2 nodes
-        as2_launch_cmd = f"ros2 launch jderobot_drones as2_default_classic_gazebo.launch.py world_file:={world_file}"
+        # Launching gzserver and Aerostack2 nodes
+        as2_launch_cmd = f"ros2 launch jderobot_drones as2_default_gazebo_sim.launch.py world_file:={world_file}"
```

Comment:
@pariaspe Should we have a separate launcher for drones for as2 harmonic drones?